testsuites/syncgateway/sgw_utilities.py

```diff
@@ -0,0 +1,20 @@
+from keywords.utils import host_for_url, log_info
+from couchbase.bucket import Bucket
+from keywords import document
+
+
+def create_docs_via_sdk(cbs_url, cbs_cluster, bucket_name, num_docs):
```

Comment:
under mobile-testkit/testsuites/ directory contains test cases python implementation only. by the nature of the method you are trying to implement, if it belongs to sync gateway functionality, it can go to mobile-testkit/keywords/SyncGateway.py, or if it's generic functionality, the entire python file can go to mobile-testkit/utilities or mobile-testkit/libraries/utilities.